### ISSUE TITLE
Disable offline compression for some scripts

### ISSUE THREAD

**Issue by navidadelpour** [django-compressor/django-compressor#1193]
Hi, I enabled the offline compression. Also, I have some scripts that need to be compressed but they heavily rely on the per-request context. For example:

```html
{% compress js inline %}
var session = {{session_from_request}}
...
{% endcompress %}
```

Is there a way to disable offline compression for this block of compression? and letting the in-request compression do the job for this part?

**Comment by bcail** [comment]
@navidadelpour Would it help to use the [COMPRESS_OFFLINE_CONTEXT](https://django-compressor.readthedocs.io/en/stable/settings.html#django.conf.settings.COMPRESS_OFFLINE_CONTEXT) setting to generate the context dynamically for each request?

**Comment by navidadelpour** [comment]
@bcail No. It would make the code very complex. the context may depends to the each view.

**Comment by diox** [comment]
I would recommend putting such code outside of the compressed blocks. If you really want to include them in the compressed blocks, then I would suggest you create a small custom template tag inheriting from django-compressor's that overrides `is_offline_compression_enabled()`.

**Comment by navidadelpour** [comment]
@diox Thanks for your response :smile::pray: . For any folks looking for the implementation:

```python
from compressor.templatetags.compress import CompressorNode, compress

class CompressorNodeForceInRequest(CompressorNode):
    def is_offline_compression_enabled(self, forced):
        return False

@register.tag
def compress_in_request(*args, **kwargs):
    result = compress(*args, **kwargs)
    return CompressorNodeForceInRequest(
        nodelist = result.nodelist,
        kind = result.kind,
        mode = result.mode,
        name = result.name
    )
    
```

```html
{% compress_in_request js inline %}
var session = {{session_from_request}}
...
{% endcompress %}
```

**Comment by navidadelpour** [comment]
@diox By the way, If you think that It would be a good feature to be able to write something like this:

```html
{% compress js inline inrequest %}
var session = {{session_from_request}}
...
{% endcompress %}
```
Or any other way of doing this, just let me know to submit a PR and add this functionality. Thanks for your great work!
